.gitignore

```diff
@@ -7,3 +7,5 @@
 /captures
 *debug*
 *release*
+/keystore.properties
+/*.jks
```

Comment:
The instructions for building a locally signed APK include putting keystore.properties and a .jks file in the root directory.  So we need Git to exclude these files.

Reply:
seems that your comment would be helpful in the code itself - at least for me as a beginner